src/forms/labels/input-error.js

```diff
@@ -70,12 +71,12 @@ const defaultProps = {
   name: '',
 }
 
-function InputError ({ error, invalid, touched, name, className, ...rest }) {
+function InputError ({ error, invalid, touched, name, className, label, ...rest }) {
   return hasInputError({ touched, invalid })
     ? <span
         id={ generateInputErrorId(name) }
         className={ classnames('error-message', className) }
-        { ...filterInvalidDOMProps(rest) }
+        { ...filterInvalidDOMProps(!label ? omit(rest, 'label') : rest) }
```

Comment:
You're already omitting `label` from `rest` on line 74, so you shouldn't need this?